From 125e6622d74a7477943c88244c48a093537b3661 Mon Sep 17 00:00:00 2001
From: Anton Kolesov <Anton.Kolesov@synopsys.com>
Date: Wed, 28 Jun 2017 13:15:46 +0300
Subject: [PATCH 13/20] gdb: Log "pc" value in "arc_skip_prologue"

Log the "pc" address upon entering "arc_skip_prologue".

gdb/ChangeLog:

	* arc-tdep.c (arc_skip_prologue): Log "pc" address.

Will be a part of GDB 11:
https://sourceware.org/git?p=binutils-gdb.git;a=commit;h=d56834cbfb7c14b2ad723c75cc56db2de3c0f0e7
---
 gdb/arc-tdep.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/gdb/arc-tdep.c
+++ b/gdb/arc-tdep.c
@@ -1493,7 +1493,7 @@
 arc_skip_prologue (struct gdbarch *gdbarch, CORE_ADDR pc)
 {
   if (arc_debug)
-    debug_printf ("arc: skip_prologue\n");
+    debug_printf ("arc: skip_prologue (pc=%s)\n", paddress (gdbarch, pc));
 
   CORE_ADDR func_addr;
   const char *func_name;
